extensions/wikia/CategoryExhibition/services/CategoryDataService.class.php

```diff
@@ -54,11 +54,9 @@ public static function getArticleCount( $sCategoryDBKey, $mNamespace = '', $nega
 
 		$db = wfGetDB( DB_SLAVE );
 
-		// Get a count of articles in a category.  Give at least a very small cache TTL
-		$query = (new WikiaSQL())->cache( 5 )
-			->SELECT( 'count(distinct page_title)' )->AS_( 'count' )
+		$query = (new WikiaSQL())
```

Comment:
I'm trying to wrap my head around what why previous version of this query seems so complicated with no apparent reason. Do you happen to know?